File: ios/RNVersaPlayer/RNVersaPlayer.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(RNVersaPlayer, NSObject)

RCT_EXTERN_METHOD(open:(NSDictionary *)options
                 resolver:(RCTPromiseResolveBlock)resolve
                 rejecter:(RCTPromiseRejectBlock)reject)

@end

File: ios/RNVersaPlayer/RNVersaPlayer.swift
import React

@objc(RNVersaPlayer)
class RNVersaPlayer: NSObject {
  var player: VersaPlayer!
  var controls: VersaPlayerControls!

    @objc(open:resolver:rejecter:)
    func open(options: NSDictionary, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
    }
}
